addons/mail/models/mail_render_mixin.py

```diff
@@ -109,9 +109,10 @@ def _update_field_translations(self, fname, translations, digest=None):
         # TBD the below check is only for model_term translations.
         # Because for model translations, super().update_field_translations will call write to check
```

Comment:
Thanks for the extra language check in `_update_field_translations` I forgot to add them back when I change some logics
Could you also remove these comments, they are not needed anymore